src/domain/worker/BusinessObject.js

```diff
@@ -0,0 +1 @@
+//business object goes here
```

Comment:
worker directory goes away, or is replaced by the persona that invokes the entity. I would probably vote to pitch it, and we would insert a persona directory of there are so many entities that we need to group them somehow.